drivers/storage/portworx/portworx_test.go

```diff
@@ -553,6 +553,109 @@ func TestValidateVsphere(t *testing.T) {
 	require.Contains(t, *cluster.Spec.CloudStorage.SystemMdDeviceSpec, DefCmetaVsphere)
 }
 
+func TestValidatePure(t *testing.T) {
+	driver := portworx{}
+	cluster := &corev1.StorageCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "px-cluster",
+			Namespace: "kube-test",
+		},
+	}
+
+	labels := map[string]string{
+		"name": pxPreFlightDaemonSetName,
+	}
+
+	// force vsphere
+	env := make([]v1.EnvVar, 1)
+	env[0].Name = "PURE_FLASHARRAY_SAN_TYPE"
+	env[0].Value = "ISCSI"
```

Comment:
Add test for another value this env can take: `FC`.

Reply:
Actually the code does not check for specific value only that PURE_FLASHARRAY_SAN_TYPE exists and that value is not empty.